recipes/adtk/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "adtk" %}
+{% set version = "0.5.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 8424fff325f9e0fc60db15d108eeef9d8e32ca2ea2d7d8c398ccb712cde1c905
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
```

Comment:
Can this be `noarch: python` instead of `skip: true  # [win]`

Reply:
Thanks. I still have this `SSL` error

```
Traceback (most recent call last):
  File "C:\projects\staged-recipes\ff_ci_pr_build.py", line 201, in <module>
    sys.exit(main())
  File "C:\projects\staged-recipes\ff_ci_pr_build.py", line 192, in main
    exit_code = int(appveyor_check_latest_pr_build(repo, pr, bld) is False)
  File "C:\projects\staged-recipes\ff_ci_pr_build.py", line 117, in appveyor_check_latest_pr_build
    data = request_json(url.format(repo=repo, total_builds=total_builds), headers=headers)
  File "C:\projects\staged-recipes\ff_ci_pr_build.py", line 46, in request_json
    with contextlib.closing(urlopen(request)) as response:
  File "C:\Python27\lib\urllib2.py", line 154, in urlopen
    return opener.open(url, data, timeout)
  File "C:\Python27\lib\urllib2.py", line 429, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 447, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 407, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1241, in https_open
    context=self._context)
  File "C:\Python27\lib\urllib2.py", line 1198, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:727)>
Command exited with code 1
```

Reply:
you don't need to worry about appveyor if azure on windows works